src/obs_inv_utils/inventory_table_factory.py

```diff
@@ -294,8 +309,10 @@ def insert_obs_meta_nceplibs_bufr_item(obs_meta_items):
     session.close()
 
 
+if(database_type.lower() == 'mysql'):
+    Base.metadata.create_all(engine)
+else:
+    create_obs_inventory_table()
+    create_cmd_results_table()
+    create_obs_meta_nceplibs_bufr_table()
```

Comment:
Looking at the diff, two question come up:
1) are you missing: `metadata.create_all(engine)` statement for the "else" case?
2) are you missing table creation statements for the `if(database_type.lower() == 'mysql')`

Reply:
So I did miss the metadata.create_all for the else case, which is needed for the sqllite engine. However, the table creation statements are correctly not included for the mysql call do to the different in how SQLalchemy builds the tables with the Base call for mysql.